tests/detect-bytemath-03/test.yaml

```diff
@@ -0,0 +1,19 @@
+requires:
+  min-version: 7
```

Comment:
`byte_math` has been around for a bit so checking for the byte_math source file could be used instead of a version:
```
requires:
   files:
       - src/detect-bytemath.c
    
```